backend/apps/subject/management/commands/import_utest_subjects.py

```
@@ -0,0 +1,461 @@
+        name = (row.get('user_name') or '').strip() or '未知'
+        phone = norm_phone(row.get('phone') or '')
+        if phone == FAKE_PHONE:
+            phone = ''
+        gender = norm_gender(row.get('sex') or '')
+        age = row.get('age')
+        city = (row.get('city') or '').strip()
+        source_channel = norm_source(row.get('source_channel') or '')
+
+        id_card_hash = sha256(id_card) if id_card else ''
+
+        # 查重：先按身份证哈希，再按手机
```

Ruff (0.15.6)

[warning] Comment contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF003)